### ISSUE TITLE
Could not instantiate ElectrumMnemonic Instance (invalid words)

### ISSUE THREAD

**Issue by anarchymachines** [Coding-Enthusiast/FinderOuter#75]
I'm trying to find a missing bip32 path, but it doesn't seem to think my old 12-word Electrum mnemonic is valid. I thought maybe it was a copy + paste issue, but I got the same result after manually typing it in. All of the words appear on the old Electrum wordlist, and "Electrum" is selected as the type (and Electrum recognizes the mnemonic as an old-style phrase and successfully allows a wallet to be recovered from it).

Are the old Electrum mnemonics not supported?

Full error message is: "Could not instantiate ElectrumMnemonic Instance. Error: Mnemonic has invalid words."

**Comment by Coding-Enthusiast** [comment]
> Are the old Electrum mnemonics not supported?

If this old algorithm [here](https://github.com/spesmilo/electrum/blob/master/electrum/old_mnemonic.py) is what you mean, then no, I never implemented it. And because the word list contains some different words, FinderOuter throws the "invalid words" exception.

**Comment by anarchymachines** [comment]
Got it. Thanks for the quick reply. And thanks for creating this! It is a lot more intuitive than other similar types of tools that I have attempted to use (and it's more useful, imo)

**Comment by Amirhossie** [comment]
How can I use finder outer

**Comment by Coding-Enthusiast** [comment]
> How can I use finder outer

It is explained in [ReadMe](https://github.com/Coding-Enthusiast/FinderOuter/blob/master/README.md#quick-guide)

**Comment by Amirhossie** [comment]
Yes I know that but I can't even find it I search the link that put in finder outer and download a zip file

**Comment by Coding-Enthusiast** [comment]
The zip files are found [here](https://github.com/Coding-Enthusiast/FinderOuter/releases). Lets say you are on Windows, you just download the [win64](https://github.com/Coding-Enthusiast/FinderOuter/releases/download/v0.18.0.0/FinderOuter-0.18.0.0-Win64SCD.zip) zip, extract it on your system and double click the exe file.

**Comment by Amirhossie** [comment]
I open file but it open so many codes what I can do ?

**Comment by Amirhossie** [comment]
It need a application?

**Comment by Amirhossie** [comment]
Did you know any one can do it for me?
